provider/provider.go

```diff
@@ -189,6 +189,12 @@ func DatabricksProvider() *schema.Provider {
 				Default:     false,
 				Description: "Create ephemeral PAT tokens also for AZ CLI authenticated requests",
 			},
+			"azure_environment": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Sensitive:   true,
```

Comment:
`Sensitive: true` might not be necessary, or do you consider it being sensitive?..

Reply:
No that was just a copy & paste mistake